src/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Baloo+Chettan+2:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+    <title>Leaderboard App</title>
+  </head>
+  <body>
+    <h1>Leaderboard</h1>
+    <main>
+      <div id="scores">
+        <div id="scores-title">
+          <h2>Recent scores</h2>
+          <button type="button">Refresh</button>
+        </div>
+        <ul></ul>
+      </div>
+      <div id="add-score">
+        <h2>Add your score</h2>
+        <form id="add-form" method="POST">
+          <input type="text" placeholder="Your name" />
+          <input type="text" placeholder="Your score" />
+          <button type="submit">Submit</button>
+        </form>
+      </div>
+    </main>
+  </body>
+</html>
```

Comment:
I can see that you have the same content in the two `index.html` files you have

<img width="288" alt="Screenshot 2021-07-27 at 23 09 07" src="https://user-images.githubusercontent.com/34798898/127234111-d8f04352-7ae6-414d-b20c-3934d31d4f4b.png">